### PR TITLE
add link to root in sidebar

### PR DESCRIPTION
**Pull request by vasya539**
- [x] merged with the latest master branch
- [x] resolved all the conflicts, PR could be merged
- [x] new code has new specs
- [x] for new models - new factories were added too
- [x] unit specs are passed
- [x] code is well-formatted, Ruby, CSS, etc
- [x] screenshots of new pages, features are added
- [x] code passed internal team code review

PS: delete your branch in github after merge http://prntscr.com/q80mvq

### CODE REVIEW

**Comment by vasya539** [comment]
image

**Comment by vasya539** [comment]
image

File: app/views/shared/account/_sidebar.html.slim [hunk 1]
```diff
@@ -24,6 +24,7 @@
         = sidebar_button({ text: 'Organizations', path: account_organizations_path, icon: 'link' })
       = sidebar_button({ text: 'Analytics', path: '#', icon: 'pie-chart' })
       = sidebar_button({ text: 'Reports', path: '#', icon: 'morph-folder' })
+      = sidebar_button({ text: 'Public Page', path: root_path, icon: 'home' })
```
Comment (igorkasyanchuk) [inline app/views/shared/account/_sidebar.html.slim]: тут якось мабуть не гуд
запхай де профіль
якусь новий пункт в менюшку додай там з розділювачем

**Comment by vasya539** [comment]
image

**Pull request merged**